src/templates/partials/coursehub-profile.twig

```diff
@@ -9,6 +9,23 @@
   <p class="f3 mb-3">{{text}}</p>
 
   <dl class="f2">
+    <div class="mb-2">
+      <dt class="d-block d-sm-inline text-bold">Pronounciation:</dt>
+      <dd class="d-inline">
+        <div class="name-spoken">" Jonathan Doe-Smith "</div>
```

Comment:
@adamfranco Are these classes like `name-spoken` selected by any css/js in D8?

Reply:
Nope, I hadn't referenced them yet. They can be removed or changed.